clients/src/test/java/org/apache/kafka/clients/ClientUtilsTest.java

```diff
@@ -96,10 +100,32 @@ public void testParseAndValidateAddressesWithReverseLookup() {
         }
     }
 
-    @Test
-    public void testInvalidConfig() {
```

Comment:
Please keep `testInvalidConfig` as it is used to test `ClientDnsLookup.forConfig` ...